tools/msmetaenhancer/test-data/sample.msp

```diff
@@ -0,0 +1,87 @@
+NAME: Hydrogen
+FORMULA: H2
+MW: 2
+CASNO: 1333740
+ID: 1
+COMMENT: NIST MS# 245692, Seq# M1
+NUM PEAKS: 2
+STDINCHI: InChI=1S/H
+SMILES: [H]
+1.0	20.98
+2.0	999.0
+
+NAME: Deuterium
+FORMULA: D2
+MW: 4
+CASNO: 7782390
+ID: 2
+COMMENT: NIST MS# 61316, Seq# M2
+NUM PEAKS: 2
+STDINCHI: InChI=1S/H2/h1H/i1+1D
+SMILES: [2H][2H]
+2.0	14.99
+4.0	999.0
+
+NAME: Methane
+FORMULA: CH4
+MW: 16
+CASNO: 74828
+ID: 3
+COMMENT: Any=100 ; NIST MS# 61313, Seq# M64
+NUM PEAKS: 6
+STDINCHI: InChI=1S/CH4/h1H4
+SMILES: C
```

Comment:
This looks like the matchms outputs from the version which didn't handle the `num peaks` field correctly - these files will be read incorrectly, matchms just silently does that. Also, please remove all metadata that is not strictly necessary for testing the tool.

Reply:
The input test file was incorrect, fixed in 8569605.